src/main.rs

```diff
@@ -294,10 +294,9 @@ impl WindowHandler for State {
         configure: WindowConfigure,
         _serial: u32,
     ) {
-        if let Some(size) = configure.new_size {
-            let size = Size::mul(size.into(), self.factor as f64);
-            self.resize(size);
-        }
+        let size = configure.new_size.unwrap_or((400, 400)).into();
```

Comment:
Should probably make this a constant?

Also I'm not sure it's a good idea to resize to 400x400 if a size was previously set already?

Reply:
Actually I think I'd do that differently.